README.md

```diff
@@ -45,26 +45,23 @@ $ helm install <release-name> <servarr-chart> \
 
 ### Values
 
-| **Parameter** | **Optional** | **Description** | **Default Value** | **Example Value** |
-|---------------|--------------|-----------------|-------------------|-------------------|
-| `global.apikey` | Required | Define a common API key that will be used to automate the entire integration. | `None` | `6d9df216-cac6-4f4a-9b25-5462026ce7be` |
-| `global.storageClassName` | Required | Cluster storage class to be used for the volume creation. | `None` | `longhorn` |
-
-<!-- Table init with some values just to give the idea on how it will be. A lot of values (almost all of them) are missing and MUST be filled in the table. -->
+Please read [README.md](./servarr/README.md) for details on how to configure the values needed for this chart.
```

Comment:
What if we change it to:

> Please, read [Chart README.md](./servarr/README.md) for details on how to configure the values needed for this chart.

the aim is to avoid any confusion with the repository readme (that they are also reading 😂)

Reply:
done!